seedbuilder/areas.ori

```diff
@@ -1628,21 +1632,25 @@ home: GinsoMiniBossDoor 544 243
 		casual Climb
 		casual ChargeJump
 		casual Bash Grenade
-		master DoubleJump
+		expert Free
 	pickup: LowerGinsoKeystone4
 		casual WallJump
 		casual Climb
```

Comment:
```suggestion
		casual Climb Health=3
```
Same for the wall jump path right above, but github is dumb and I can't make a suggestion for it